pages/express-relay/integrate-as-searcher.mdx

```diff
@@ -1,6 +1,7 @@
-import { Callout, Tabs, Steps } from "nextra/components";
+import { Cards, Card } from "nextra/components";
+import CodeIcon from "../../components/icons/CodeIcon";
 
-# How to Integrate Express Relay as a Searcher
+# Integrating as a Searcher
```

Comment:
Integrate as a Searcher.

It's a good practice to keep the heading and index heading names same. 

Or just leave it to 'How to Integrate Express Relay as a Searcher'